docs/dev/table/streaming/joins.md

```diff
@@ -222,4 +222,131 @@ By definition of event time, [watermarks]({{ site.baseurl }}/dev/event_time.html
 forward in time and discard versions of the build table that are no longer necessary because no incoming row with
 lower or equal timestamp is expected.
 
+Join with a Temporal Table
+--------------------------
+
+**Notes:** This is only supported in blink planner.
```

Comment:
We should also mention that users cannot use any arbitrary tables for temporal join but need to use a table backed by a `LookupableTableSource`. Can people also perform other operations on top of this table before like projection? I guess no, right?

Reply:
You are right. We can't perform other operations on a `LookupableTableSource`. But if the table source also implements `StreamTableSource`, it can support other operations. 

I will add this notes.